Process.c

```diff
@@ -408,6 +414,17 @@ void Process_toggleTag(Process* this) {
    this->tag = this->tag == true ? false : true;
 }
 
+bool Process_isNew(const Process* this, const time_t now) {
+    if (this->starttime_ctime > 0) {
+        return (now - this->starttime_ctime) <= 5;
```

Comment:
Same as the comment on the tomb_time calculation (not time jump safe as not using monotonic clock and safe time calculation).

Reply:
Better have an internal field in htop marking when htop first saw each process (using some monotonic timer).